### ISSUE TITLE
Having histograms show proportions and percentages

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#5765]
The main histogram dialogue is setup to allow this option.  But it doesn't yet do it.  I don't quite understand the 3 different options.
The default is to give counts and that's fine.
Then there are fractions, which could be in percentages - which don't work
Then there are "scaled fractions" but without any option for a scale.
It should be a simple fix to give the fractions - then (for now at least) don't have the scaled fractions.
The site [here](https://stackoverflow.com/questions/3695497/show-instead-of-counts-in-charts-of-categorical-variables) gives the explanation.

When I use fractions the code generated by R-Instat  with my correction is here:
```
# Code generated by the dialog, Histogram Plot

mydata <- data_book$get_data_frame(data_name="mydata")
last_graph <- ggplot2::ggplot(data=mydata, mapping=ggplot2::aes(x=wd, fill=Recode2,y=(..count..)/sum(..count..))) + ggplot2::geom_histogram(bins=12) + theme_grey() + ggplot2::scale_y_continuous(labels=scales::percent_format())
data_book$add_graph(graph_name="last_graph", graph=last_graph, data_name="mydata")
data_book$get_graphs(data_name="mydata", graph_name="last_graph")
rm(list=c("last_graph", "mydata"))

```
The correction is to add y=(..count..)/sum(..count..) into the aesthetic, see above.

I hope this is easy to do.

**Comment by rdstern** [comment]
@Wycklife  is questioning whether it is actually working and his example - on skype - looks ok.  First let's check we are using the same dialogue - namely histogram and not bar chart.  Then the simple histogram with fractions (on the data he uses) gives me this with 6 bins:
![image](https://user-images.githubusercontent.com/11226469/76992881-d83fb300-6943-11ea-81f7-42373ad4a009.png)

When I add the village as a factor and set the bins back to the default I have this dialogue:
![image](https://user-images.githubusercontent.com/11226469/76992968-ff968000-6943-11ea-85ec-dfacd8aeb6f8.png)

and this result:

![image](https://user-images.githubusercontent.com/11226469/76993043-2654b680-6944-11ea-9fa3-6bef17ea094e.png)

I still don't see - with histograms - how to get the result he showed me - and I could get with the script window.

**Comment by Wycklife** [comment]
@africanmathsinitiative/developers Could someone please try out this. I seem not to be getting any error as @rdstern did.

**Comment by shadrackkibet** [comment]
I have checked this and the above bug identified by @rdstern exists. Code seems to be mixed up. The issue is with using top options (Histogram, Density and Frequency Polygon) at different instances. I also noticed that the default for the **percentage** checkbox is not set properly, it is always checked when you reopen or use different stat options which is also causing the mess. Check how the code for the different stat options is set to fix the problem. 

On the scaled fractions, I think what we need is the correct terminology. What is referred to as scaled fractions here is dividing the counts by the maximum count  which is achieved by `y=stat(count/max(count)) `
